docker-compose.yml

```
@@ -5,53 +5,139 @@ services:
+      ${WEBAPP_DOMAIN}: ${WEBAPP_HOST_IP}
+      ${KEYCLOAK_DOMAIN}: ${KC_HOST_IP}
     ports:
-      - '8080:5000'
+      - '8081:5000'
     environment:
-      DB_USER: ${DB_USER}
-      DB_PASS: ${DB_PASS}
-      DB_NAME: ${DB_NAME}
+      PG_DB_HOST: ${PG_DB_HOST}
+      PG_DB_USER: ${PG_DB_USER}
+      PG_DB_PASS: ${PG_DB_PASS}
+      PG_DB_NAME: postgres
       INSTANCE_CONNECTION_NAME: ${INSTANCE_CONNECTION_NAME}
-      CORS_DOMAIN: ${CORS_DOMAIN}
-      GOOGLE_CLIENT_ID: ${GOOGLE_CLIENT_ID}
-      LOGGING_LEVEL: ${LOGGING_LEVEL}
-      RSU_REST_ENDPOINT: ${RSU_REST_ENDPOINT}
-      COUNT_DB_NAME: ${COUNT_DB_NAME}
-      TIMEZONE: ${TIMEZONE}
-      WZDX_API_KEY: ${WZDX_API_KEY}
-      WZDX_ENDPOINT: ${WZDX_ENDPOINT}
+
+      MONGO_DB_URI: ${MONGO_DB_URI}
+      MONGO_DB_NAME: ${MONGO_DB_NAME}
+
+      COUNTS_MSG_TYPES: ${COUNTS_MSG_TYPES}
+      COUNTS_DB_TYPE: ${COUNTS_DB_TYPE}
+      COUNTS_DB_NAME: ${COUNTS_DB_NAME}
+      GOOGLE_APPLICATION_CREDENTIALS: '/google/gcp_credentials.json'
+
       BSM_DB_NAME: ${BSM_DB_NAME}
       SSM_DB_NAME: ${SSM_DB_NAME}
       SRM_DB_NAME: ${SRM_DB_NAME}
+
+      WZDX_API_KEY: ${WZDX_API_KEY}
+      WZDX_ENDPOINT: ${WZDX_ENDPOINT}
+
+      CORS_DOMAIN: ${CORS_DOMAIN}
+      KEYCLOAK_ENDPOINT: http://${KEYCLOAK_DOMAIN}:8084/
+      KEYCLOAK_REALM: ${KEYCLOAK_REALM}
+      KEYCLOAK_API_CLIENT_ID: ${KEYCLOAK_API_CLIENT_ID}
+      KEYCLOAK_API_CLIENT_SECRET_KEY: ${KEYCLOAK_API_CLIENT_SECRET_KEY}
+
+      CSM_EMAIL_TO_SEND_FROM: ${CSM_EMAIL_TO_SEND_FROM}
+      CSM_EMAIL_APP_USERNAME: ${CSM_EMAIL_APP_USERNAME}
+      CSM_EMAIL_APP_PASSWORD: ${CSM_EMAIL_APP_PASSWORD}
+      CSM_EMAILS_TO_SEND_TO: ${CSM_EMAILS_TO_SEND_TO}
+      CSM_TARGET_SMTP_SERVER_ADDRESS: ${CSM_TARGET_SMTP_SERVER_ADDRESS}
+      CSM_TARGET_SMTP_SERVER_PORT: ${CSM_TARGET_SMTP_SERVER_PORT}
+
+      TIMEZONE: ${TIMEZONE}
+      LOGGING_LEVEL: ${API_LOGGING_LEVEL}
+    volumes:
+      - ${GOOGLE_APPLICATION_CREDENTIALS}:/google/gcp_credentials.json

```

Are we expecting a directory to hold this credentials file?

---

Looks like we have sample file in `resources/google` should this volume be updated to reflect that location?

---

Looks like @drewjj  is attaching a secret to the pod with the following [volume mount](https://dev.azure.com/SOC-OIT/CDOT/_git/cdot-cv-gcp?path=/GKE/cv-manager/templates/cv-manager-api.yaml&version=GBkc-helm-updates&_a=contents):
volumeMounts:
  - name: cv-manager-service-key
    mountPath: /home/secret

---

> Looks like we have sample file in `resources/google` should this volume be updated to reflect that location?

Are you thinking that we should be updated the sample.env variable: "GOOGLE_APPLICATION_CREDENTIALS" to default to the sample file? We don't want the real file to be committed to Git, so I think that's the thought behind having it default to a new file.

---

Ack, I was reading the mount backwards for some reason, must not have had enough coffee. Looks like we have an appropriate dummy value in the sample.env file.